scripts/kpm.cmd

```diff
@@ -2,7 +2,7 @@
 SETLOCAL
 SET ERRORLEVEL=
 
-"%~dp0klr" --appbase "%CD%" %K_OPTIONS% --lib "%~dp0lib\Microsoft.Framework.PackageManager" Microsoft.Framework.PackageManager %*
+"%~dp0dotnet" --appbase "%CD%" %K_OPTIONS% --lib "%~dp0lib\Microsoft.Framework.PackageManager" Microsoft.Framework.PackageManager %*
```

Comment:
Also we should simplify this script now. It's:

`dotnet "%~dp0lib\Microsoft.Framework.PackageManager.dll" %*`